pkgs/poetry/update.nix

```diff
@@ -1,6 +1,5 @@
 let
-  sources = import ../../nix/sources.nix;
-  pkgs = import sources.nixpkgs {
+  pkgs = import <nixpkgs> {
```

Comment:
@adisbladis Is there a better way to do this that doesn't depend on special path syntax?

Reply:
I will rewrite this as a flake application in a separate PR and then rebase this PR on that one.